containers/newtask/withStateHandlers.ts

```diff
@@ -0,0 +1,55 @@
+import { compose, withStateHandlers } from 'recompose';
+import { Colors } from '../../colors/Colors';
+
+const withTitleHandlers = withStateHandlers(
+  { title: '' },
+  {
+    onChangeTitle: state => (text: string) => ({
+      title: text
+    })
+  }
+);
+
+const withDescriptionHandlers = withStateHandlers(
```

Comment:
What do you think about refactoring this hocs to use `withState`?

Reply:
But we need to use withHandlers to implement the state handler, what is the advantage of use withStateHandlers?

Reply:
`withStateHandlers` enables you to have *multiple* handlers so is generally used when you have some complex state like an object, and want different handlers to update different parts of it.
`withState` only enables you to have *1* state updater so it's usually used when your state is just one variable.